### ISSUE TITLE
Wireguard doesn't work from my machine to the grid but works with DO

### ISSUE THREAD

**Issue by OmarElawady** [threefoldtech/zos#1442]
There were some articles discussing MTU configurations but this needs a closer look from the node.

**Comment by OmarElawady** [comment]
The largest packet sent or received is 190 bytes on wire so probably not a MTU problem. I tried again and it seems that this also happens on DO.

More Info:
The machine's wireguard sends an initialization packet to DO, DO wireguard receives it and replies with a packet. This packet gets dropped and this is why the handshake fails. Every subsequent packet from DO or the machine gets dropped. Turning the wireguard down and up on the machine repeats the same scenario (first outgoing packet goes through and the rest dropped).

Tried also from the internet package and the behavior is the same 95% of the time, downing and upping multiple times results in a successful handshake every now and then (both on DO and the grid).

Digital ocean tcpdump:
![image](https://user-images.githubusercontent.com/13040543/138711410-6a550c49-feea-497c-a3d9-4edadb53c0c9.png)

My machine tcpdump:
![image](https://user-images.githubusercontent.com/13040543/138711533-95c67959-47ad-426f-8151-ced7c0ad040c.png)
